hack/verify-govet.sh

```diff
@@ -24,7 +24,7 @@ function govet_blacklist_contains() {
 }
 
 for test_dir in $(os::util::list_go_src_dirs); do
-	if ! result="$(go tool vet -shadow=false -printfuncs=Info,Infof,Warning,Warningf "${test_dir}" 2>&1)"; then
+	if ! result="$(go vet -shadow=false -printfuncs=Info,Infof,Warning,Warningf "${test_dir}/..." 2>&1)"; then
```

Comment:
I don't think this will work without go install before for go <=1.9

Reply:
I don't think we need to support Go <1.9 on master